opensrp-anc/src/test/java/org/smartregister/anc/library/presenter/PreviousContactTestsPresenterTest.java

```diff
@@ -78,14 +78,14 @@ public void setUp() {
     @Test
     public void testLoadPreviousContactsTest() throws ParseException, IOException {
         PowerMockito.mockStatic(AncLibrary.class);
-        PowerMockito.when(AncLibrary.getInstance()).thenReturn(AncLibrary);
+        PowerMockito.when(org.smartregister.anc.library.AncLibrary.getInstance()).thenReturn(AncLibrary);
         //Create dummy yml configuration settings
         Iterable<Object> testsRuleObjects = new ArrayList<>();
 
         populateYamlConfiRules((ArrayList<Object>) testsRuleObjects);
 
         PowerMockito.when(AncLibrary.getPreviousContactRepositoryHelper()).thenReturn(previousContactRepositoryHelper);
-        PowerMockito.when(AncLibrary.getInstance().readYaml(FilePathUtils.FileUtils.PROFILE_LAST_CONTACT_TEST)).thenReturn(testsRuleObjects);
+        PowerMockito.when(org.smartregister.anc.library.AncLibrary.getInstance().readYaml(FilePathUtils.FileUtils.PROFILE_LAST_CONTACT_TEST)).thenReturn(testsRuleObjects);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.anc.library.AncLibrary.getInstance' due to existing import 'org.smartregister.anc.library.AncLibrary'](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=4809639)